File: checkTables.py
import sqlite3

conn = sqlite3.connect('./data/database.db')
cursor = conn.cursor()

# Get all tables
cursor.execute("SELECT name FROM sqlite_master WHERE type='table';")
tables = cursor.fetchall()

print("Tables in database:")
for table in tables:
    print(f"  - {table[0]}")
    
# For each table, show its structure
for table in tables:
    table_name = table[0]
    print(f"\n{table_name} structure:")
    cursor.execute(f"PRAGMA table_info({table_name})")
    columns = cursor.fetchall()
    for col in columns:
        print(f"  - {col[1]} {col[2]}")
    
    # Count rows
    cursor.execute(f"SELECT COUNT(*) FROM {table_name}")
    count = cursor.fetchone()[0]
    print(f"  Row count: {count}")

conn.close()

File: check-tables.js
const { getDb, initializeDb } = require('./src/utils/sqliteDb.js');

async function checkTables() {
  await initializeDb();
  const db = await getDb();
  
  console.log('=== Existing Tables ===');
  const tables = db.prepare("SELECT name FROM sqlite_master WHERE type='table'").all();
  tables.forEach(table => console.log('- ' + table.name));
  
  console.log('\n=== Files Table Sample ===');
  const files = db.prepare('SELECT * FROM files LIMIT 5').all();
  files.forEach(file => {
    console.log(`- ${file.file_path} (status: ${file.status || 'N/A'})`);
  });
}

checkTables();